flow.c

```diff
@@ -19,6 +19,9 @@
 #include "socket.h"
 #include "thread.h"
 #include "stats.h"
+#ifdef WITH_TCPDIRECT
+#include "tcpdirect.h"
```

Comment:
tcpdevmem.h

But also, to support udmabuf, you may want to split into tcpdevmem.h, tcpdevmem-cuda.h & tcpdevmem-udmabuf.h, maybe.